ddl/ddl_worker.go

```diff
@@ -348,6 +347,13 @@ func (d *ddl) handleDDLJobQueue() error {
 			// no job now, return and retry get later.
 			return nil
 		}
+		if binlogStartTS != 0 {
```

Comment:
If `binlogStartTS != 0` is ture, but finishDDLJob is failed. In this case can it write post DDL binlog?